querybook/server/tasks/disable_scheduled_docs.py

```diff
@@ -117,12 +117,12 @@ def check_task_no_impression_for_n_days(
 ):
     doc_id = task_dict["kwargs"]["doc_id"]
 
-    if not disable_config.skip_if_no_impression_with_export:
+    if disable_config.skip_if_no_impression_with_export:
         task_have_export = len(task_dict["kwargs"].get("exports", [])) > 0
         if task_have_export:
             return False
 
-    if not disable_config.skip_if_no_impression_but_non_select:
+    if disable_config.skip_if_no_impression_but_non_select:
```

Comment:
@czgu could you help to look at this? I'm a bit confused about the actual meaning of these two configs. should it disable or not when they are true.
skip_if_no_impression_with_export
skip_if_no_impression_but_non_select